packages/plugin-ext/package.json

```diff
@@ -50,7 +50,8 @@
     {
```

Comment:
You can add `frontendElectron` in another entry point to rebind something from `frontend` of first entry point:
```json
"theiaExtensions": [
    {
      "backend": "lib/plugin-ext-backend-module",
      "backendElectron": "lib/plugin-ext-backend-electron-module",
      "frontend": "lib/plugin-ext-frontend-module"
    },
    {
      "frontendElectron": "lib/plugin-ext-frontend-electron-module"
    }
  ]
```